test/framework/cluster_proxy.go

```diff
@@ -206,6 +213,32 @@ func (p *clusterProxy) GetClientSet() *kubernetes.Clientset {
 	return cs
 }
 
+func (p *clusterProxy) GetPodInformer(ctx context.Context) cache.Informer {
+	if p.cache == nil {
+		c, err := cache.New(p.GetRESTConfig(), cache.Options{
+			Scheme: p.scheme,
+			Mapper: p.GetClient().RESTMapper(),
+		})
+		Expect(err).ToNot(HaveOccurred(), "Failed to create controller-runtime cache")
+
+		go func() {
+			defer GinkgoRecover()
+			Expect(c.Start(ctx)).NotTo(HaveOccurred())
+		}()
+
+		p.cache = &c
+	}
+
+	if p.podInformer == nil {
```

Comment:
Please double-check but as far as I'm ware cache internally caches informers and doesn't recreate them

Reply:
```
// Get will create a new Informer and add it to the map of InformersMap if none exists.  Returns
// the Informer from the map.
```

Reply:
Ack, https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/cache/informer_cache.go#L148